src/main/amp/config/alfresco/subsystems/Transformers/default/remote-jod-transformers.properties

```diff
@@ -49,6 +49,11 @@ content.transformer.complex.RemoteJODConverter.Html2Pdf.pipeline=RemoteJODConver
 content.transformer.complex.RemoteJODConverter.Html2Pdf.extensions.html.pdf.supported=true
 content.transformer.complex.RemoteJODConverter.Html2Pdf.extensions.html.pdf.priority=50
 
+
+content.transformer.RemoteJODConverter.extensions.tiff.pdf.priority=30
+content.transformer.RemoteJODConverter.extensions.tiff.pdf.supported=true
+content.transformer.RemoteJODConverter.extensions.tiff.pdf.maxSourceSizeKBytes=100000
```

Comment:
Can you move this line to the other limits, more towards the top of the file?

100 MB is also a very large size, compared to the supported size for other documents.
For a default, it's maybe better to have something smaller like 10 MB (`10240`, the same size as doc to pdf)